index.html

```diff
@@ -34,6 +34,11 @@
       rel="stylesheet"
     />
     <title>LazuLight 1st Anniversary</title>
+    <script
```

Comment:
This script is used by Twitter embeds.  It looks for `blockquote` elements with class `twitter-tweet` and converts them into interactive embeds.